src/index.html

```diff
@@ -11,6 +11,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="searchBar" data-qa="big">
+      <label>
+        <button type="submit" class="searchBar__button">
+          <img
+            src="./images/Search.svg"
+            alt="SearchIcon"
+            class="searchBar__img"
+          >
+        </button>
+        <input
+          type="text"
+          class="searchBar__input"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+        >
+      </label>
+    </form>
+    <form class="searchBar searchBar--small" data-qa="small">
+      <label>
+        <button type="submit" class="searchBar__button searchBar--small__button">
+          <img
+            src="./images/Search.svg"
```

Comment:
Consider using a more descriptive alt text for accessibility. For example, `Search icon` or `Magnifying glass icon`. ♿